src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -411,6 +488,9 @@ private void SelectionOnCollectionChanged(object sender, NotifyCollectionChanged
            CreateGroupCommand.RaiseCanExecuteChanged();
            AddToGroupCommand.RaiseCanExecuteChanged();
            UngroupCommand.RaiseCanExecuteChanged();
+           ComputeRunStateOfTheNodeCommand.RaiseCanExecuteChanged();
```

Comment:
If we are not doing the workspace and application level stuff, can we get rid of this command?